### DISCUSSION TITLE
ci for create and gitpod customisation

### DISCUSSION THREAD

**Discussion by ahmedali8** [paulrberg/foundry-template#4] [Ideas]
Hi @paulrberg, thanks for merging [PR #137](https://github.com/paulrberg/hardhat-template/pull/137) in the Hardhat template repo.

Please let me know if it's a go ahead from your side to re-implement that very feature in this template. I'll be more than happy to open a PR. Thanks.

**Comment by PaulRBerg** [comment]
Hi @ahmedali8, of course, happy to add the same functionality in this repo. Thank you.

**Comment by PaulRBerg** [comment]
Going to close this discussion since this repo now supports this CI workflow that runs on creating a copy of the template.
